aws/v12.1.1/README.md

```diff
@@ -0,0 +1,19 @@
+# :zap: Giant Swarm Release v12.1.1 for AWS :zap:
+
+<< Add description here >>
```

Comment:
Is this some some kind of devctl sorcery?

Reply:
goddamnit i forgot to fill out the readme. yes this was generated with devctl